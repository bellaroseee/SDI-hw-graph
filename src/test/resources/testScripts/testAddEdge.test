# Test that Edges can be added to the graph

# Create a graph
CreateGraph graph1

# Add 2 pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add 2 edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2

# Create another graph
CreateGraph graph2

# Add 1 pair of nodes
AddNode graph2 n1
AddNode graph2 n2

# Add 1 edge
AddEdge graph2 n1 n2 e1

# Add 1 node
AddNode graph2 n3

# Add another edge
AddEdge graph2 n1 n3 e2