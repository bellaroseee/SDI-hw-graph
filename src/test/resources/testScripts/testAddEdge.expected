# Test that Edges can be added to the graph

# Create a graph
created graph graph1

# Add 2 pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add 2 edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1

# Create another graph
created graph graph2

# Add 1 pair of nodes
added node n1 to graph2
added node n2 to graph2

# Add 1 edge
added edge e1 from n1 to n2 in graph2

# Add 1 node
added node n3 to graph2

# Add another edge
added edge e2 from n1 to n3 in graph2